rooms.js

```diff
@@ -653,7 +652,7 @@ class GlobalRoom {
 	}
 	onConnect(user, connection) {
 		let initdata = '|updateuser|' + user.name + '|' + (user.named ? '1' : '0') + '|' + user.avatar + '\n';
-		connection.send(initdata + this.configRankList + '\n' + this.formatListText);
+		connection.send(initdata + (Config.serverid !== 'showdown' ? this.configRankList : '') + '\n' + this.formatListText);
```

Comment:
Would not hardcode it this way.

One possibility is a `Config` option that determines whether or not we're using the default Config settings.